### PR TITLE
ISSUE-63: Delete added Regex pattern from database

### PR DESCRIPTION
**Pull request by alimuhammadahmer**
This is a pull request which contains Ruuter and Resql endpoints to delete a regex record.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,5 +11,19 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
 
 https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.yml
 
+```
+```
```
Comment (turnerrainer) [inline README.md:15]: Please add a sample curl request.

File: README.md [hunk 2]
```diff
@@ -11,5 +11,19 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+``` 
+
+### Delete added Regex pattern from database
+
+https://github.com/buerokratt/Data-Anonymizer/issues/63
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/delete_regex.sql
+
+```
+```
```
Comment (turnerrainer) [inline README.md:24]: Please add a sample curl request.

File: README.md [hunk 3]
```diff
@@ -11,5 +11,19 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
...
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/GET/delete_regex.yml
+
 ```
 ```
```
Comment (turnerrainer) [inline README.md:29]: Please add a sample curl request.

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (alimuhammadahmer) [reply README.md]: Done

File: DSL.Resql/delete_regex.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+insert into Regex_Entity (id, regex, entity, active) select id, regex, entity, False from Regex_Entity where id = :id ORDER BY created_at DESC limit 1;
```
Comment (turnerrainer) [inline DSL.Resql/delete_regex.sql:1]: No newline at end of file

**Review by turnerrainer — CHANGES_REQUESTED** [review]
No newline at end of file, otherwise OK

File: DSL.Resql/delete_regex.sql (hunk 4 above)
Comment (alimuhammadahmer) [reply DSL.Resql/delete_regex.sql:1]: Done

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**